Block/Info/Hpp.php

```diff
@@ -40,6 +40,31 @@ public function isPaymentSelectionOnAdyen()
         return $this->_adyenHelper->getAdyenHppConfigDataFlag('payment_selection_on_adyen');
     }
 
+    /**
+     * Get all Banktransfer related data
+     *
+     * @return array
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getBankTransferData()
+    {
+        $result = [];
+        if(!empty($this->getInfo()->getOrder()->getPayment()->getAdditionalInformation('bankTransfer.owner'))) {
```

Comment:
the same

Reply:
order can't never have no payment in this stage but I will add it for extra defensive programming